src/app/components/BookDetail/styles.ts

```diff
@@ -1,32 +1,78 @@
 import { StyleSheet } from 'react-native';
+import { ALTO, CERULEAN, MONZA, TUNDORA, WHITE } from '@constants/colors';
 import { SHADOW } from '@constants/commonStyles';
-import { TUNDORA, WHITE } from '@constants/colors';
+import { FONT_BOLD, FONT_HEAVY, FONT_LIGHT } from '@constants/fontWeights';
 
 export default StyleSheet.create({
   container: {
-    borderRadius: 5,
     backgroundColor: WHITE,
-    marginHorizontal: 10,
-    flexDirection: 'row',
-    marginTop: 10,
-    paddingVertical: 15,
-    paddingHorizontal: 20,
+    margin: 20,
+    borderRadius: 5,
     ...SHADOW
   },
+  infoContainer: {
+    flexDirection: 'row',
+    marginTop: 15,
+    marginBottom: 30,
+    paddingLeft: 20
+  },
   image: {
-    height: 60,
-    width: 40
+    height: 105,
+    width: 69
+  },
+  rentButton: {
+    marginHorizontal: 30,
+    height: 44,
+    borderRadius: 22,
+    backgroundColor: ALTO,
+    marginBottom: 22,
+    justifyContent: 'center'
+  },
+  rentButtonText: {
+    textAlign: 'center',
+    fontWeight: FONT_HEAVY,
+    color: WHITE
   },
-  textContainer: {
-    marginHorizontal: 20
+  wishButton: {
+    marginHorizontal: 30,
+    height: 44,
+    borderRadius: 22,
+    borderWidth: 1,
+    borderColor: CERULEAN,
+    marginBottom: 10,
+    justifyContent: 'center'
+  },
+  wishButtonText: {
+    textAlign: 'center',
+    color: CERULEAN,
+    fontWeight: FONT_HEAVY
+  },
```

Comment:
There are some similar styles between the two buttons and inside texts. Define one style object and then you can use the spread operator, for example:
```
const BUTTON = {
    marginHorizontal: 30,
    height: 44,
    borderRadius: 22,
    justifyContent: 'center'
};

...
wishButton: {
  ...BUTTON,
  extra styles...
}
...
```
Same with texts